packages/store/ts/config/storeConfig.ts

```diff
@@ -71,7 +71,7 @@ const zShorthandSchemaConfig = zFieldData.transform((fieldData) => {
 
 export const zSchemaConfig = zFullSchemaConfig.or(zShorthandSchemaConfig);
 
-export type ResolvedSchema<
+type ResolvedSchema<
```

Comment:
not used anywhere and the name overlaps with our experimental config types, so I removed this export